### ISSUE TITLE
JSONDecodeError upon empty response body

### ISSUE THREAD

**Issue by ariel-m-s** [daleal/zum#48]
A [`JSONDecodeError`](https://docs.python.org/3/library/json.html) (from the built-in `json` library) is raised for HTTP responses with an empty body (which is, naturally, not a JSON). I don’t think this is the expected behavior.
```
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I guess the expected behavior should be to return an empty string.

**Comment by daleal** [comment]
Yes, that is definitely **not** the expected behavior. I think that maybe if the request body doesn't exist (and the text property is empty), Zum should show the response code of the request. Or maybe showing nothing is the right behavior. In any case, the CLI shouldn't fail on no body.

**Comment by ariel-m-s** [comment]
Would it be better to add a condition (`if response.text`) o just try to parse is as JSON and catch the exception?

**Comment by daleal** [comment]
I think that a combination may be the correct way to handle it. Some `response.text` might not be JSON-valid but still contain text. In that case, the text should be shown.
